keyboards/kradoindustries/kousa/info.json

```diff
@@ -0,0 +1,155 @@
+{
+    "keyboard_name": "Kousa-TKL", 
+    "manufacturer": "Krado Industries",
+    "url": "https://www.kradoindustries.com/", 
+    "maintainer": "Krado Industries", 
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x12D0",
```

Comment:
This vendor ID is already assigned: 
https://www.the-sz.com/products/usbid/index.php?v=0x12D0&p=&n=

In general, something above 0x4000 should be used, or a service such as pid.codes should be used for the VID and PID.

Reply:
Didn't know I had to check that site. Thanks.